theseus_gui/src/App.vue

```diff
@@ -392,6 +451,55 @@ command_listener((e) => {
   }
 }
 
+.failure {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  background-color: #16181c;
```

Comment:
you should never use raw colors, and use the variables provided by omorphia. see other parts of the code

Reply:
The issue here was the colors were loading weirdly at this point (prob. because something doesnt get initialized before the app initializes?)
Either way, fixed now